pyogrio/raw.py

```diff
@@ -384,12 +409,22 @@ def open_arrow(
     >>>     for table in reader:
     >>>         geometries = shapely.from_wkb(table[meta["geometry_name"]])
 
+    Or without directly returning a pyarrow object:
+
+    >>> with open_arrow(path) as source:
+    >>>     meta, stream = source
+    >>>     reader = pa.RecordBatchReader.from_stream(stream)
+    >>>     for table in reader:
+    >>>         geometries = shapely.from_wkb(table[meta["geometry_name"]])
```

Comment:
```suggestion
    >>>         geometries = shapely.from_wkb(table[meta["geometry_name"] or "wkb_geometry"])
```

since `geometry_name` is often blank like in many of our tests